.scalafmt.conf

```diff
@@ -0,0 +1,12 @@
+version=2.7.5
```

Comment:
There is a new version we might as well go all the way to 3.0.0

Reply:
You are right, @osleonard. That work was done before 3.0.0 has been released. I'll update this PR soon.